browser/go/conversion-test/conversion_test.go

```diff
@@ -0,0 +1,451 @@
+// +build !js
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os/exec"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/chromedp/cdproto/runtime"
+	"github.com/chromedp/chromedp"
+	"github.com/stretchr/testify/require"
+)
+
+var testCases []string
+
+func TestBrowserConversions(t *testing.T) {
+	// Declare a context that will be used for all child processes, servers, and
+	// other goroutines.
+	ctx, cancel := context.WithTimeout(context.Background(), 180*time.Second)
+	ctx, _ = chromedp.NewContext(ctx, chromedp.WithErrorf(t.Errorf))
+	defer cancel()
+
+	buildForTests(t, ctx)
+
+	// Register the test cases that should be logged.
+	registerContractEventTest()
+	registerGetOrdersResponseTest("emptyOrderInfo", 0)
+	registerGetOrdersResponseTest("oneOrderInfo", 1)
+	registerGetOrdersResponseTest("twoOrderInfos", 2)
+	registerOrderEventTest("EmptyContractEvents", 0)
+	registerOrderEventTest("ExchangeFillContractEvent", 1)
+	registerSignedOrderTest("NullAssetData")
+	registerSignedOrderTest("NonNullAssetData")
```

Comment:
Nit: Why are some names capitalized, and others aren't?

Reply:
No reason. I'll make them all lowercase (my preference)

Reply:
* I am actually going to capitalize all of them because it makes more sense to leave the ContractEvent types as capitalized.